### PR TITLE
Update Glossary For zh_Hans

### PR DESCRIPTION
**Pull request by bestony**

### CODE REVIEW

**Comment by felixxm** [comment]
Thanks for the report, however, translations are handled at [Transifex](https://docs.djangoproject.com/en/dev/internals/contributing/localizing/#translations).

**Comment by bestony** [comment]
@felixxm Hi, you can see the topic at google groups at https://groups.google.com/forum/#!topic/django-i18n/hnKsUg1CSvA

I ask Claude How to Make new translation and can i request a pull request.

Maybe you can merge for me or tell me what i should tell you.

**Comment by felixxm** [comment]
@bestony Sorry, I confused repository. Forget about my comment :+1:

File: translations/zh_Hans/LC_MESSAGES/glossary.po [hunk 1]
```diff
@@ -52,17 +54,17 @@ msgid ""
...
 "implementation of a common idiom or pattern found in view development."
 msgstr ""
-"一个高阶的 :term:`view` 函数针对开发中常见的习惯用法或模式, 提供了一个抽象/通"
-"用的实现."
+"高阶的 :term:`view` 函数针对开发中常见的习惯用法或模式，提供了一种抽象/通用的"
+"实现。"
 
 msgid "See :doc:`/topics/class-based-views/index`."
-msgstr "参见 :doc: `/topics/class-based-views/index` 。"
+msgstr "参见 :doc: `/topics/class-based-views/index`。"
```
Comment (claudep) [inline translations/zh_Hans/LC_MESSAGES/glossary.po]: I think there shouldn't be a space between `:doc:` and `

Comment (bestony) [reply translations/zh_Hans/LC_MESSAGES/glossary.po]: @claudep  OK, I will the the translator of this. Is here a style book for translator?

Comment (bestony) [reply translations/zh_Hans/LC_MESSAGES/glossary.po]: I will let the translator to review all translations.

and i will close this issue until we review all code .

Comment (bestony) [reply translations/zh_Hans/LC_MESSAGES/glossary.po]: will , i just want a code style ,just like there shouldn't be a space between `:doc:` and `

i think the django docs not  markdown style.

Comment (claudep) [reply translations/zh_Hans/LC_MESSAGES/glossary.po]: Ah, the syntax is reST: http://docutils.sourceforge.net/rst.html

Comment (bestony) [reply translations/zh_Hans/LC_MESSAGES/glossary.po]: thx

**Pull request closed without merging**